app/uk/gov/hmrc/helptosavefrontend/controllers/EligibilityCheckController.scala

```diff
@@ -157,7 +157,7 @@ class EligibilityCheckController @Inject() (val messagesApi:             Message
                                                         htsContext: HtsContextWithNINO): Result = error.value match {
     case Left(e) ⇒
       logger.warn(e, htsContext.nino)
-      InternalServerError
+      InternalServerError(internalServerError())
```

Comment:
can we move this `InternalServerError` into `internalServerError()` so that we can simply call `internalServerError()` everywhere instead of `InternalServerError(internalServerError())` ?